README.md

```diff
@@ -13,7 +13,9 @@ TorchServe now enforces token authorization enabled and model API control disabl
 
 TorchServe is a flexible and easy-to-use tool for serving and scaling PyTorch models in production.
 
-Requires python >= 3.8
+Requires: 
+- python >= 3.8
+- Java >= 17
```

Comment:
Is Java a hard dependency? curious why it wasn't here originally

Reply:
It is but it's installed via install scripts. I'll remove this

Reply:
fixed with 8e4d24cc63a640fff86d8a775dd9ed530c57d3d6